### ISSUE TITLE
No display when connecting a Meizu PRO 6

### ISSUE THREAD

**Issue by fishhhhh** [Genymobile/scrcpy#709]
Scrcpy: v1.10
Device: Meizu Pro 6
Android version: 7.1.1
ROM: Flyme 7.3.0.0A
Host: Windows 10 1803

```
D:\Program Files (x86)\scrcpy-win64-v1.10>scrcpy
INFO: scrcpy 1.10 <https://github.com/Genymobile/scrcpy>
D:\Program Files (x86)\scrcpy-win64-v1.10\scrcpy-server.jar: 1 file pushed. 6.2 MB/s (22546 bytes in 0.003s)
INFO: Initial texture: 1080x1920
```
and hold on here... no response, no display window

it's the adb logcat result
```
08-09 17:42:03.442   397 15335 E DrmManagerService(Native): isProtectedCallAllowed in uid 1013 spid 402
08-09 17:42:03.442   397 15335 E DrmManagerService(Native): isProtectedCallAllowed out uid 1013 spid 402, result 1
08-09 17:42:03.442   397 15335 E DrmManagerService(Native): isProtectedCallAllowed DrmMtkUtil::isTrustedClient uid 1013 spid 402, result 1
08-09 17:42:03.442   397 15335 D DrmManager(Native): pid = 402
08-09 17:42:03.442   397 15335 D DrmManager(Native): pid_str = 402
08-09 17:42:03.442   397 15335 D DrmCtaPlugIn: onOpenDecryptSession() [4450], fd = 9, offset = 0, length = 7995
08-09 17:42:03.442   397 15335 E DrmCtaPlugIn: [ERROR]onOpenDecryptSession - The key is not set by user
08-09 17:42:03.443   397 15335 E DrmMtkUtil/DrmUtil: parseDcf: not dcf type, dcf version value [79]
08-09 17:42:03.443   397 15335 E DrmMtkPlugIn: onOpenDecryptSession() : failed to parse dcf file.
08-09 17:42:03.447   401  1405 D MediaExtractor: JB +MediaExtractor::Create
08-09 17:42:03.449   397 15336 E DrmManagerService(Native): isProtectedCallAllowed in uid 1013 spid 402
08-09 17:42:03.449   397 15336 E DrmManagerService(Native): isProtectedCallAllowed out uid 1013 spid 402, result 1
08-09 17:42:03.449   397 15336 E DrmManagerService(Native): isProtectedCallAllowed DrmMtkUtil::isTrustedClient uid 1013 spid 402, result 1
08-09 17:42:03.449   397 15336 D DrmManager(Native): pid = 401
08-09 17:42:03.449   397 15336 D DrmManager(Native): pid_str = 401
08-09 17:42:03.449   397 15336 D DrmCtaPlugIn: onOpenDecryptSession() [6000], fd = 9, offset = 0, length = 7995
08-09 17:42:03.449   397 15336 E DrmCtaPlugIn: [ERROR]onOpenDecryptSession - The key is not set by user
08-09 17:42:03.449   397 15336 E DrmMtkUtil/DrmUtil: parseDcf: not dcf type, dcf version value [79]
08-09 17:42:03.449   397 15336 E DrmMtkPlugIn: onOpenDecryptSession() : failed to parse dcf file.
08-09 17:42:03.453   401  1405 D MPEG2PSExtractor: +SniffMPEGPS in
08-09 17:42:03.453   401  1405 D MPEG2PSExtractor: ********************* parsePackHeader can't find pack header start code *********************
08-09 17:42:03.459   401  1405 I MediaExtractor: Autodetected media content as 'application/ogg' with confidence 0.20
08-09 17:42:03.461   401  1405 E OggExtractor: failed to read 27 bytes at offset 0x00000000001f3b, got 0 bytes
08-09 17:42:03.461   401  1405 D MediaExtractor: JB -MediaExtractor::Create
08-09 17:42:03.468   397  1403 E DrmManagerService(Native): isProtectedCallAllowed in uid 1013 spid 402
08-09 17:42:03.468   397  1403 E DrmManagerService(Native): isProtectedCallAllowed out uid 1013 spid 402, result 1
08-09 17:42:03.468   397  1403 E DrmManagerService(Native): isProtectedCallAllowed DrmMtkUtil::isTrustedClient uid 1013 spid 402, result 1
08-09 17:42:03.469   397  1403 D DrmManager(Native): pid = 402
08-09 17:42:03.469   397  1403 D DrmManager(Native): pid_str = 402
08-09 17:42:03.469   397  1403 D DrmCtaPlugIn: onOpenDecryptSession() [4872], fd = 9, offset = 0, length = 8074
08-09 17:42:03.469   397  1403 E DrmCtaPlugIn: [ERROR]onOpenDecryptSession - The key is not set by user
08-09 17:42:03.469   397  1403 E DrmMtkUtil/DrmUtil: parseDcf: not dcf type, dcf version value [79]
08-09 17:42:03.469   397  1403 E DrmMtkPlugIn: onOpenDecryptSession() : failed to parse dcf file.
08-09 17:42:03.474   401   507 D MediaExtractor: JB +MediaExtractor::Create
08-09 17:42:03.476   397   615 E DrmManagerService(Native): isProtectedCallAllowed in uid 1013 spid 402
08-09 17:42:03.476   397   615 E DrmManagerService(Native): isProtectedCallAllowed out uid 1013 spid 402, result 1
08-09 17:42:03.476   397   615 E DrmManagerService(Native): isProtectedCallAllowed DrmMtkUtil::isTrustedClient uid 1013 spid 402, result 1
08-09 17:42:03.476   397   615 D DrmManager(Native): pid = 401
08-09 17:42:03.476   397   615 D DrmManager(Native): pid_str = 401
08-09 17:42:03.476   397   615 D DrmCtaPlugIn: onOpenDecryptSession() [8049], fd = 9, offset = 0, length = 8074
08-09 17:42:03.476   397   615 E DrmCtaPlugIn: [ERROR]onOpenDecryptSession - The key is not set by user
08-09 17:42:03.477   397   615 E DrmMtkUtil/DrmUtil: parseDcf: not dcf type, dcf version value [79]
08-09 17:42:03.477   397   615 E DrmMtkPlugIn: onOpenDecryptSession() : failed to parse dcf file.
08-09 17:42:03.484   401   507 D MPEG2PSExtractor: +SniffMPEGPS in
08-09 17:42:03.484   401   507 D MPEG2PSExtractor: ********************* parsePackHeader can't find pack header start code *********************
08-09 17:42:03.491   401   507 I MediaExtractor: Autodetected media content as 'application/ogg' with confidence 0.20
08-09 17:42:03.494   401   507 E OggExtractor: failed to read 27 bytes at offset 0x00000000001f8a, got 0 bytes
08-09 17:42:03.494   401   507 D MediaExtractor: JB -MediaExtractor::Create
08-09 17:42:06.756   403 20205 D libc-netbsd: [getaddrinfo]: netid=298; mark=917802; ai_flags=1024; ai_family=0
08-09 17:42:06.757   403 20205 D libc-netbsd: [getaddrinfo]: hostname=yp-data.meizu.com; servname=(null); netid=298; mark=917802; ai_flags=1024; ai_family=0
08-09 17:42:06.757   403 20205 D libc-netbsd: default dns: query_ipv6=0, query_ipv4=1
08-09 17:42:06.772   403 20205 D libc-netbsd: res_queryN succeed
08-09 17:42:06.774 12518 20198 D libc-netbsd: getaddrinfo: yp-data.meizu.com get result from proxy gai_error = 0
08-09 17:42:06.835   403 20208 D libc-netbsd: [getaddrinfo]: netid=298; mark=917802; ai_flags=1024; ai_family=0
08-09 17:42:06.836   403 20208 D libc-netbsd: [getaddrinfo]: hostname=scan.call.f.360.cn; servname=(null); netid=298; mark=917802; ai_flags=1024; ai_family=0
08-09 17:42:06.836   403 20208 D libc-netbsd: default dns: query_ipv6=0, query_ipv4=1
08-09 17:42:06.854   403 20208 D libc-netbsd: res_queryN succeed
08-09 17:42:06.856 22359 20204 D libc-netbsd: getaddrinfo: scan.call.f.360.cn get result from proxy gai_error = 0
08-09 17:42:08.349   403 20253 D libc-netbsd: [getaddrinfo]: netid=298; mark=298; ai_flags=1024; ai_family=0
08-09 17:42:08.349   403 20253 D libc-netbsd: [getaddrinfo]: hostname=apilocate.amap.com; servname=(null); netid=298; mark=298; ai_flags=1024; ai_family=0
08-09 17:42:08.350   403 20253 D libc-netbsd: default dns: query_ipv6=0, query_ipv4=1
08-09 17:42:08.363   403 20253 D libc-netbsd: res_queryN succeed
08-09 17:42:08.363 12699 20246 D libc-netbsd: getaddrinfo: apilocate.amap.com get result from proxy gai_error = 0
08-09 17:42:08.491   403 20254 D libc-netbsd: [getaddrinfo]: netid=298; mark=298; ai_flags=1024; ai_family=0
08-09 17:42:08.492   403 20254 D libc-netbsd: [getaddrinfo]: hostname=aider.meizu.com; servname=(null); netid=298; mark=298; ai_flags=1024; ai_family=0
08-09 17:42:08.492   403 20254 D libc-netbsd: default dns: query_ipv6=0, query_ipv4=1
08-09 17:42:08.502   403 20254 D libc-netbsd: res_queryN succeed
08-09 17:42:08.502 12699 20245 D libc-netbsd: getaddrinfo: aider.meizu.com get result from proxy gai_error = 0
```

Just for your reference, or for other who met the same issue.

**Comment by rom1v** [comment]
Probably #151

**Comment by fishhhhh** [comment]
> Probably #151

So this problem has not been fixed yet? I use the latest version of Scrcpy and Flyme,and the screen record is OK.
```
Scrcpy: v1.10
Device: Meizu Pro 6
Android version: 7.1.1
ROM: Flyme 7.3.0.0A
Host: Windows 10 1803
```

**Comment by xjboss** [comment]
I has same problem
Scrcpy: v1.10
Device: Meizu M6 Note
Android version: 7.1.2
ROM: Flyme 7.0.1.1A
Host: Windows 10 1903

**Comment by fishhhhh** [comment]
I'm using the latest version and the problem is still unresolved
Scrcpy: v1.11
Device: Meizu Pro 6
Android version: 7.1.1
ROM: Flyme 7.3.0.0A
Host: Windows 10 1903

**Comment by rom1v** [comment]
Please execute `adb logcat -c`, start scrcpy. Then:

    adb logcat -d -f adb.log

Then post the whole `adb.log` file here.

**Comment by fishhhhh** [comment]
[adb.log](https://github.com/Genymobile/scrcpy/files/3873091/adb.log)
It's the log
when I run scrcpy,the log is empty.
until i use my phone,the log has content.

**Comment by rom1v** [comment]
> when I run scrcpy,the log is empty.

That does help debugging :/

What is the exact output you get in the scrcpy console with v1.11?

If you record, you get the same problem I guess:

```bash
scrcpy -Nr file.mkv
# unplug to interrupt
```

What if you use a lower definition?

```
scrcpy -m800
```

**Comment by fishhhhh** [comment]
when I rerun scrcpy, the log has content.
[adb(1).log](https://github.com/Genymobile/scrcpy/files/3873365/adb.1.log)
```
D:\software\scrcpy-win64-v1.11>scrcpy
INFO: scrcpy 1.11 <https://github.com/Genymobile/scrcpy>
D:\software\scrcpy-win64-v1.11\scrcpy-server: 1 file pushed. 2.0 MB/s (25454 bytes in 0.012s)
INFO: Initial texture: 1080x1920
```

```
D:\software\scrcpy-win64-v1.11>scrcpy -m800
INFO: scrcpy 1.11 <https://github.com/Genymobile/scrcpy>
D:\software\scrcpy-win64-v1.11\scrcpy-server: 1 file pushed. 2.2 MB/s (25454 bytes in 0.011s)
INFO: Initial texture: 448x800
```
scrcpy hold on here.

**Comment by rom1v** [comment]
OK. There is not enough information. :(

I think it is necessary to [debug](https://github.com/Genymobile/scrcpy/blob/master/DEVELOP.md#debug-the-server) the server while running to find out where it hangs. I suspect that `MediaCodec` does not produce any data, but I don't know why.

**Comment by fishhhhh** [comment]
The issue is resolved in the v1.12.1.
